contracts/Response.sol

```diff
@@ -82,20 +82,47 @@ contract Response {
   }
 
   function questionState(uint qIdx) internal view returns (QuestionState t) {
+    // packedState is all of the data about the question x`packed into a single integer
     uint packedState = s.qUint(qIdx, PACKED_STATE);
+
+    // the twitter user id is the last 64 bits of the packedState
     t.twitterUserId = uint64(packedState);
+
+    // packedstate is shifted right 64 bits
     packedState >>= 64;
+
+    // the tweet id is the last 64 bits
     t.tweetId = uint64(packedState);
+
+    // packedstate is shifted right 64 bits
     packedState >>= 64;
+
+    // the tweet creation date is the last 32 bits
     t.createdAt = uint32(packedState);
+
+    // packedstate is shifted right 32 bits
     packedState >>= 32;
+
+    // the supporter count is the last 32 bits
     t.supporterCount = uint32(packedState);
+
+    // packedstate is shifted right 32 bits
     packedState >>= 32;
+
+    // QUESTION: is my understanding correct?
+    // if remaining bits are all 0 then answered = false
+    // if any remaining bits are 1 then answered = true
```

Comment:
`packedState & 1` will return only last bit. So, it the last bit is 1 then answered = true.